### PR TITLE
Impro 1263 more references in optical flow

### PR DESCRIPTION
**Pull request by fionaRust**
Addresses IMPRO-1263

Just a few more references in docstrings whilst I was looking at them.

### CODE REVIEW

File: lib/improver/nowcasting/optical_flow.py [hunk 1]
```diff
@@ -242,6 +242,11 @@ def _make_subboxes(self, field):
                 **weights** (np.ndarray):
                     1D numpy array containing weights values associated with
                     each listed box.
+
+        References:
+            Bowler, N., Pierce, C. and Seed, A. 2004: Development of a
+            precipitation nowcasting algorithm based upon optical flow
+            techniques. Journal of Hydrology, 288, 74-91.
```
Comment (bayliffe) [inline lib/improver/nowcasting/optical_flow.py]: This is referenced at the top of the plugin, do we need the repeat? You could consider moving it out of the __init__ doc string and into the generic plugin doc string, but I don't think we need repeats throughout.

File: lib/improver/nowcasting/optical_flow.py [hunk 2]
```diff
@@ -319,6 +324,9 @@ def smooth(self, field, radius, method='box'):
         Smoothing method using a square ('box') or circular kernel.  Kernel
         smoothing with a radius of 1 has no effect.
 
+        Smoothing with the "box" argument is equivalent to the method
+        in equation 7 in Bowler et al. 2004.
```
Comment (bayliffe) [inline lib/improver/nowcasting/optical_flow.py]: I think the reference is clear without the repeat in the function specific doc-string.

File: lib/improver/nowcasting/optical_flow.py [hunk 3]
```diff
@@ -330,6 +338,11 @@ def smooth(self, field, radius, method='box'):
         Returns:
             smoothed_field (np.ndarray):
                 Smoothed data on input-shaped grid
+
+        References:
+            Bowler, N., Pierce, C. and Seed, A. 2004: Development of a
+            precipitation nowcasting algorithm based upon optical flow
+            techniques. Journal of Hydrology, 288, 74-91.
```
Comment (bayliffe) [inline lib/improver/nowcasting/optical_flow.py]: See above.

File: lib/improver/nowcasting/optical_flow.py (hunk 1 above)
Comment (fionaRust) [reply lib/improver/nowcasting/optical_flow.py]: I've moved all the references to the top level plugin documentation

**Pull request merged**